testsuite/features/secondary/srv_timezone.feature

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
```

Comment:
```suggestion
@scope_visualization
```

Reply:
Add a scope, so we can run this feature in a small group, if necessary.

Reply:
done